scripts/setup.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/bash
+
+## Declare variables
+# VARS:
+DOCKER_IMAGE="proximax/proximax-sirius-chain:v0.9.0-bullseye" # This info should be grapped from github
+DEFAULT_PATH="/mnt/siriuschain/"
+SIRIUS_CHAIN_VERSION="release-v0.9.0"
+P2P_PACKAGE="public-mainnet-peer-package-$SIRIUS_CHAIN_VERSION.tar.gz"
+DOWNLOAD_URL="https://github.com/proximax-storage/xpx-mainnet-chain-onboarding/releases/download/$SIRIUS_CHAIN_VERSION/$P2P_PACKAGE"
+FRIENDLYNAME="mainnet-${HOSTNAME%%.*}"
+
+
+## Prerequisite
+# check whether Docker is installed
+if [ -x "$(command -v docker)" ]; then
+    echo "docker is installed"
+else
+    echo "Docker is not installed"
+    exit 1
+fi
+
+if [ -x "$(command -v docker-compose)" ]; then
+    echo "docker-compose is installed"
+else
+    echo "docker-composse is not installed"
+    exit 1
+fi
```

Comment:
This might probably fail due to Docker moving Compose to a Plugin which is called via `docker compose` instead.

Reply:
added check by running `docker compose version` command in script